### PR TITLE
add ADPLAN

### PR DESCRIPTION
**Pull request by matzkoh**
ADPLAN is Analytics tool that is popular in Japan.

Users example:

- https://www.mfr.co.jp/
- https://www.acom.co.jp/
- https://www.sekisuihouse.co.jp/
- https://www.u-can.co.jp/
- https://www.shiseido.co.jp/
- http://www.yakult.co.jp/
- https://deagostini.jp/
- https://www.lotte.co.jp/
- https://www.mouse-jp.co.jp/
- https://www.baitoru.com/

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -328,6 +328,17 @@
...
     },
+    "ADPLAN": {
+      "cats": [
+        10
+      ],
+      "icon": "ADPLAN.png",
+      "js": {
+        "_adp._order": ""
+      },
+      "script": "(\\.adplan7\\.com/)|(?!/o)..\\.advg\\.jp/\\;version:\\1?7:",
```
Comment (jvoisin) [inline src/apps.json]: Wouldn't it be simpler to write something like that instead:

```json
"script": [
  "\\.adplan7\\.com/",
  "(?!/o)..\\.advg\\.jp/\\;version:\\1?7:"
]
```

I'm a bit afraid that the second pattern will lead to false-positives :/

**Comment by matzkoh** [comment]
Updated.

**Comment by jvoisin** [comment]
You just reverted your modifications :/

**Comment by matzkoh** [comment]
@jvoisin 
https://travis-ci.org/AliasIO/Wappalyzer/builds/409337755
Because Validation failed.
What should I do?

**Comment by jvoisin** [comment]
Check the error message in the CI:

> Error: The version string references more capture groups than there are in the pattern! Remove any incorrect back references from the version string and/or add the missing capture groups to the pattern.
ADPLAN: script: ^https?://(?!o\.)\w+\.advg\.jp/\;version:\1?7:

Apparently, the CI doesn't like your negative group.

**Comment by matzkoh** [comment]
```
"script": [
  "foo",
  "bar\\;version:\\1?7:"
]
```

Same error.

The above patterns makes one capture group implicitly (`foo`).

So this version detections are valid and works fine on the extension.

I know the validator says "no capture group found on `bar`".

But It's difficult for me to contribute to `validate-regex` ;(

**Comment by matzkoh** [comment]
Ok I found better pattern.

It works on local extension and CI passed.

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**